Features/inclusief-eind-status.feature

```diff
@@ -0,0 +1,73 @@
+# language: nl
+
+Functionaliteit: inclusief eind status
+
+  Bij endpoints die standaard huidige (actueel zonder eind status) gegevens leveren,
+  kan middels een parameter inclusiefEindStatus ook de actuele gegevens worden opgevraagd. 
+  Actuele gegevens zijn de gegevens die nu geldig zijn ongeacht de status van het object voorkomen.
+
+  Deze parameter kan niet worden gebruikt i.c.m. tijdreis parameters, zie https://github.com/lvbag/BAG-API/blob/master/Features/tijdreizen.feature of huidig parameter, zie https://github.com/lvbag/BAG-API/blob/master/Features/huidig.feature.
```

Comment:
Het endpoint waar `inclusiefEindStatus` toegevoegd gaat worden heeft geen `huidig`, `geldigOp` en `beschikbaarOp` parameter.
De scenario's hieronder beschrijven daarom meer dan wat in de praktijk kan voorkomen? Of is de verwachting dat de `inclusiefEindStatus` parameter breder ingezet gaat worden?